wpimath/src/main/java/edu/wpi/first/math/geometry/Rotation2d.java

```diff
@@ -46,20 +46,41 @@ public class Rotation2d
    */
   public static final Rotation2d kCW_Pi_2 = new Rotation2d(-Math.PI / 2);
 
+  /**
+   * A preallocated Rotation2d representing a counterclockwise rotation by −90° (−π/2 rad).
```

Comment:
Would it be better to say "a clockwise rotation by 90" versus "a counterclockwise rotation by -90"?  That would be more consistent with the constant name.